demodoctrine/views/templates/admin/create.html.twig

```diff
@@ -7,7 +7,7 @@
  * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
  *#}
 
-{% extends 'PrestaShopBundle:Admin:layout.html.twig' %}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
```

Comment:
```suggestion
{% extends '@PrestaShopCore/Admin/layout.html.twig' %}
```

Reply:
hmm, now that I'm thinking about this change, we know it requires a new method of overriding views:

`{% extends '@!PrestaShop/Admin/layout.html.twig' %}`

or

`{% extends '@PrestaShopCore/Admin/layout.html.twig' %}` - but this is only from PS 8.0

https://devdocs.prestashop-project.org/8/modules/concepts/templating/admin-views/#override-templates

We had a report recently that overriding by `{% extends '@PrestaShopCore/Admin/layout.html.twig' %}` may cause an infinite loop on PrestaShop 8

Reply:
Thank you @kpodemski I've update to override views with :

`{% extends '@!PrestaShop/Admin/layout.html.twig' %}`

If we use this version `{% extends '@PrestaShopCore/Admin/layout.html.twig' %}` the module will not be compatible with PS 1.7.

Reply:
Haha 😄 @kpodemski @thomasnares this obliges us to think about what should example modules demonstrate: how to use the last stable version ? How to have a module that is compatible with multiple versions?

Reply:
Yeah, having a separate repo would allow us to create tags for specific versions, on the other hand:

- for module developers who sell on marketplaces - it is better to have a wider compatibility range
- for agencies - it is important to work on the recent version

I'm in favor of option 1, ideally, we could have a comment in the code or something indicating that "Hey, since 8.0 you can do X instead of Y, enjoy!"

Reply:
updated ! :)